merlin/models/tf/inputs/embedding.py

```diff
@@ -381,6 +385,14 @@ def Embeddings(
     infer_embeddings_ensure_dim_multiple_of_8 : bool, optional
         Ensures that the inferred embedding sizes are rounded up to the next multiple of 8,
         for better performance for embedding looks on GPUs. By default True
+    non_trainable : Optional[Dict[str, bool]] = None
```

Comment:
Removing _non_ prefix for consistency based on previous comment I think worth doing now that it's a Dict.